docs/user/user/howto/real-time-events/real-time-events.md

```diff
@@ -0,0 +1,37 @@
+# "Real-Time" Events
+
+## Defining "Real-Time" Glean Events
+
+"Real-Time" in the context of Glean [Events](../../../reference/metrics/event.md) directly relates to configuring Glean to send each event in an [Events Ping](../../pings/events.md) as soon as it is recorded.
```

Comment:
"This minimizes the time between instrumentation and reporting, but doesn't describe how quickly received data is made available for querying."